xml/admin_ceph_gateway.xml

```diff
@@ -3757,7 +3757,8 @@ rgw_zone=us-west</screen>
   <para>
    You can use the &haproxy; load balancer to distribute all requests across
    multiple &ogw; back-end servers. Refer to
-   <link xlink:href="https://www.suse.com/documentation/sle-ha-12/book_sleha/data/sec_ha_lb_haproxy.html"/>
+   <link
+    xlink:href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-guide/#sec-ha-lb-haproxy"/>
```

Comment:
you refer to 15sp1 section here, is that intentional?

Reply:
Nope, accident. Will fix.